### PR TITLE
DOC: Move pandas_development_faq from wiki to doc #30232

### PR DESCRIPTION
**Pull request by AdrianMastronardi**
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

**Comment by gfyoung** [comment]
cc @pandas-dev/pandas-core

File: doc/source/development/pandas_development_faq.rst [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+Pandas Development FAQ
+======================
+
+.. contents:: Table of contents:
+   :local:
+
+Purpose
+=======
+
+Based on https://github.com/pydata/pandas/pull/4404#issuecomment-22864665 this
```
Comment (datapythonista) [inline doc/source/development/pandas_development_faq.rst]: This link is outdated, we're in the pandas-dev organization now. Also, you can make it a link with rst syntax.

File: doc/source/development/pandas_development_faq.rst [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+
+from the ``pandas`` directory.
+
+Travis
+------
+
+**Q:** Where do I need to change the settings in my GitHub configuration and/or
+Travis configuration for the Travis to start builds from my fork?
+
+**A:** To be filled out.
```
Comment (datapythonista) [inline doc/source/development/pandas_development_faq.rst]: Probably better to remove this question

File: doc/source/development/pandas_development_faq.rst [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+This is one of the nice aspects of
+`distributed version control <http://en.wikipedia.org/wiki/Distributed_revision_control>`_.
+
+Documentation
+-------------
+
+**Q:** Does Travis build documentation?
+
+**A:** Currently, no. There are some issues surrounding Sphinx error reporting.
+We are investigating ways to solve this problem.
```
Comment (datapythonista) [inline doc/source/development/pandas_development_faq.rst]: Remove this, it's outdated

File: doc/source/development/pandas_development_faq.rst [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+--------
+
+* What is a typical workflow on my local fork?
+* Shall I work in a virtualenvironment?
+* Shall I work in a virtualenvironment and then copy my changes over into a
+  clean local fork of my own repo?
+
+**Q:** Who will be responsible for evaluating my PR?
+
+**A:** Technically, anyone with push rights to the ``pydata/pandas`` can
```
Comment (datapythonista) [inline doc/source/development/pandas_development_faq.rst]: Pydata - > pandas-dev

**Review by datapythonista — COMMENTED** [review]
Some comments, but personally I don't think anything in this file adds much value, I'd simply remove it from the wiki without adding to the documentation

File: doc/source/development/pandas_development_faq.rst (hunk 1 above)
Comment (AdrianMastronardi) [reply doc/source/development/pandas_development_faq.rst]: I fix it.

File: doc/source/development/pandas_development_faq.rst (hunk 3 above)
Comment (AdrianMastronardi) [reply doc/source/development/pandas_development_faq.rst]: I fix it.

File: doc/source/development/pandas_development_faq.rst (hunk 2 above)
Comment (AdrianMastronardi) [reply doc/source/development/pandas_development_faq.rst]: I fixed it.

File: doc/source/development/pandas_development_faq.rst (hunk 4 above)
Comment (AdrianMastronardi) [reply doc/source/development/pandas_development_faq.rst]: I fixed it.

**Comment by AdrianMastronardi** [comment]
@datapythonista thanks for your feedback. Meanwhile we wait for other contributors to decide whether keep this doc or remove it, I fixed the errors you pointed out.

**Comment by datapythonista** [comment]
Thanks @AdrianMastronardi.

If nobody objects I'll be removing the wiki page and closing this PR soon. If anyone wants to keep anything of that wiki page, please let me know.

**Comment by WillAyd** [comment]
I think fine to get rid of the wiki page

**Comment by datapythonista** [comment]
Deleted the wiki page, closing this.

**Comment by datapythonista** [comment]
Thanks @AdrianMastronardi, and sorry for the late decision (after you did all the work). Please feel free to ping me if you want to move any other page, and you're unsure if the content is still relevant.

**Pull request closed without merging**